frontend/system/src/Admin/Configuration/CategoryConfigTab.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  AppBarButtonsPortal,
+  AppBarContentPortal,
+  DropdownMenu,
+  DropdownMenuItem,
+  LoadingButton,
+  useConfirmationModal,
+} from '@common/components';
+import { useTranslation } from '@common/intl';
+import {
+  createTableStore,
+  DataTable,
+  DeleteIcon,
+  PlusCircleIcon,
+  TableProvider,
+  useColumns,
+  useTableStore,
+} from '@common/ui';
+import { useEditModal } from '@common/hooks';
+import { CategoryOptionEditModal } from './CategoryOptionEditModal';
+
+// TODO: this type should come from gql codegen types
+type CategoryOption = {
+  id: string;
+  label: string;
+  value: string;
+};
+
+type CategoryConfigTabProps = {
+  data: CategoryOption[];
+  category: string;
+};
+
+const CategoryConfigTabComponent = ({
```

Comment:
The name `CategoryConfigTab` confused me a little bit, as we have categories for `Drug` and `Consumable`
Maybe `ListConfigTab`? Can't think of a perfect name :)

Reply:
The `Categories` are `types` in the dgraph but that's also confusing.
If we're considering changing a lot about the dgraph, we could make category the thing like `drug`, `unit` etc?
Maybe that's another approach...

Reply:
Gone with `OptionList`... not sure I love that name either but at least I don't think its in conflict with anything else 😅  [a0c6cbe](https://github.com/msupply-foundation/unified-codes/pull/462/commits/a0c6cbe20854227bf55890d6b876cde11fd1e1c8)